app/Exceptions/Auth/InvalidUserDataException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Hedonist\Exceptions\Auth;
+
+use Throwable;
+
+class InvalidUserDataException extends \Exception
+{
+    public function __construct(
+        string $message = "Sorry, looks like your data is corrupted. Please, contact our customer support",
```

Comment:
It is ok, but consider to use smth like factory methods in exception for message generating. Example:
```
public static function create(): self
{
  return new self("Sorry, looks like your data is corrupted. Please, contact our customer support");
}
```